bin/git-standup

```diff
@@ -149,13 +149,26 @@ if [[ $# -gt 0 ]]; then
 fi
 
 AUTHOR=${AUTHOR:="$(git config user.name)"}
-SINCE=${SINCE:=yesterday}
-UNTIL=${UNTIL:=today}
 FETCH_LAST_COMMIT=${FETCH_LAST_COMMIT:=false}
 MAXDEPTH=${MAXDEPTH:=2}
 GIT_PRETTY_FORMAT="%Cred%h%Creset - %s %Cgreen(%cd) %C(bold blue)<%an>%Creset $GIT_PRETTY_FORMAT"
 GIT_DATE_FORMAT=${GIT_DATE_FORMAT:=relative}
 
+# Handle config of implicit week
+if [[ -z "$SINCE" ]] && [[ -z "$UNTIL" ]] && [[ -n "$(git config --get git-extras.standup-implicit-week)" ]]; then
```

Comment:
```suggestion
if [[ -z "$SINCE" ]] && [[ -z "$UNTIL" ]] && [[ -n "$(git config --get git-extras.standup.implicit-week)" ]]; then
```
would be better?